kedro/framework/session/session.py

```diff
@@ -78,14 +78,22 @@ class KedroSession:
     - Use `KedroSession(session_id=<id>)` to instantiate an existing session with a given
     ID.
 
+
+
     Example:
     ::
 
         >>> from kedro.framework.session import KedroSession
-        >>>
+        >>> from kedro.framework.startup import bootstrap_project
+        >>> from pathlib import Path
+
+        >>> # If you are creating session outside of a kedro project, i.e. not using
+        >>> # `kedro run` or `kedro jupyter`. You need to run `bootstrap_project` to
+        >>> # let Kedro find your configuration.
+        >>> bootstrap_project(<your-pyproject.toml-directory>)
         >>> with KedroSession.create("<your-kedro-project-package-name>") as session:
```

Comment:
```suggestion
        >>> metadata = bootstrap_project(Path("<project_root>"))
        >>> with KedroSession.create(metadata.package_name) as session:
```
@deepyaman recently made all references to the project root in documentation look like this I think.